lib/types/utils.ts

```diff
@@ -2,11 +2,11 @@ export type WithOptionalSys<Type extends { sys: unknown }> = Omit<Type, 'sys'> &
   sys?: Type['sys']
 }
 
-export interface Link<Type = string> {
+export interface Link<LinkType = string, Type = string> {
```

Comment:
Can we make `Link` the default?

```suggestion
export interface Link<LinkType = string, Type extends string = 'Link`> {
```

And why not making `Type` a fixed value as in cma.js 
https://github.com/contentful/contentful-management.js/blob/ebc7af28d184db031fad8179b24cab2d4fbc5bb6/lib/common-types.ts#L70-L76

Reply:
Oh... great - let's keep it like this